pom.xml

```diff
@@ -21,6 +21,9 @@
 
     <java.abi>1.7</java.abi>
     <mesos.version>0.20.0</mesos.version>
+    <log4j-over-slf4j.version>1.7.10</log4j-over-slf4j.version>
+    <jcl-over-slf4j.version>1.7.10</jcl-over-slf4j.version>
```

Comment:
`slf4j` components are released together with the same versions.  It would be cleaner to have a single `slf4j.version` property.